guides/v2.3/comp-mgr/cli/upgrade-with-plugin.md

```diff
@@ -0,0 +1,195 @@
+---
+group: software-update-guide
+title: (Experimental) Command-line upgrade using the Magento composer root plugin
+functional_areas:
+  - Upgrade
+---
+
+<!-- Topic variables
+{% capture ce %}{{site.data.var.ce}}{% endcapture %}
+{% capture ee %}{{site.data.var.ee}}{% endcapture %}
+-->
+
+{: .bs-callout-warning}
+This upgrading scenario is still under development and is published here to make it available for the Community to test.
+Do not use it on production.
+
+The [`magento/composer-root-update-plugin`][custom composer plugin] Composer plugin resolves changes that need to be made to the root project `composer.json` file before updating to a new Magento product requirement.
```

Comment:
This link is not rendering properly in HTML.

Reply:
fixed